### PR TITLE
Project CRUD

### PR DESCRIPTION
**Pull request by SradhaBhavsar**
Project controller CRUD

### CODE REVIEW

File: app/controllers/admin/projects_controller.rb [hunk 1]
```diff
@@ -19,9 +19,69 @@ def show
...
+
+   
+  def create
+    @project = Project.new(project_params)
+    if @project.save
+      render json: {
+      data: ActiveModelSerializers::SerializableResource.new(@project, serializer: ProjectSerializer),
+      message: ['Project created successfully'],
+      status: 200,
+      type: 'Success'
```
Comment (saikrishnapujari97) [inline app/controllers/admin/projects_controller.rb:35]: use rails status codes  http://www.railsstatuscodes.com/
Example : render json: { success :true}, status: :ok

File: app/controllers/admin/projects_controller.rb [hunk 2]
```diff
@@ -19,9 +19,69 @@ def show
     }
   end
 
+  def new
```
Comment (saikrishnapujari97) [inline app/controllers/admin/projects_controller.rb:22]: we don't required this api.

File: app/controllers/admin/projects_controller.rb [hunk 3]
```diff
@@ -19,9 +19,69 @@ def show
...
+      status: 200,
+      type: 'Success'
+    }
+    else
+      render json: {errors: @project.errors, status: 400}
+    end
+  end
+
+  def edit
+    @project = Project.find_by(id: params[:id])
```
Comment (saikrishnapujari97) [inline app/controllers/admin/projects_controller.rb:43]: This should be show 
def show

**Review by saikrishnapujari97 — CHANGES_REQUESTED** [review]

File: app/controllers/admin/projects_controller.rb (hunk 1 above)
Comment (SradhaBhavsar) [reply app/controllers/admin/projects_controller.rb:35]: @saikrishnapujari97 I think, here the status should be 201. status: :created

**Pull request merged**